nservicebus/upgrades/asp-6to7.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Azure Storage Persistence Upgrade Version 6 to 7
+summary: Instructions on how to upgrade Azure Storage Persistence Version 6 to 7.
+tags:
+ - upgrade
+ - migration
+related:
+- nservicebus/azure-storage-persistence
+- nservicebus/upgrades/asp-saga-deduplication
+---
+
+## Saga de-duplication patch
+Upgrades from NServiceBus.Azure v6.2.3 or lower will need to apply the [saga de-duplication patch](/nservicebus/upgrades/asp-saga-deduplication.md) before completing the remainder of these upgrade steps.
+
+## Changing Nuget Packages and namespaces
+Upgrading requires removing the NServiceBus.Azure Nuget package from the project and adding the NServiceBus.Persistence.AzureStorage package in it's place.
+
+Once the packages have been changed, any use of the `NServiceBus.Azure` namespace needs to be replaced with `NServiceBus.Persistence.AzureStorage`
+
+## New Configuration API
+In Versions 6 and below the Azure Storage Persistence was configured using XML configuration sections called `AzureSubscriptionStorageConfig`, `AzureSagaPersisterConfig`, and `AzureTimeoutPersisterConfig`. These sections have been removed in favor of a more granular, code based configuration API.
```

Comment:
i would like to see some before after snippets that show what to replace the config sections with